### PR TITLE
Job-consumer container: run as fedora user

### PR DESCRIPTION
**Pull request by danielbeardsley**
Other services end up running as appropriate users for their service, but since our docker image doesn't force a user, it was defaulting to root.

This meant that if this process created any dirs or files, they'd only be accessible to root. This was breaking some features that allowed downloading the results or logs of background jobs.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -72,6 +72,7 @@ services:
 
   job-consumer:
     build: ./docker
+    user: 1000:1000
```
Comment (djmetzle) [inline docker-compose.yml:75]: The `docker` build here is on buster?
https://github.com/iFixit/pim-community-standard/blob/8fc949b31e6efcbd005585a4192d674d3eaf6c26/docker/Dockerfile#L1

Comment (djmetzle) [reply docker-compose.yml:75]: That doesnt really matter.

**Review by djmetzle — APPROVED** [review]
CR :bust_in_silhouette:

**Comment by danielbeardsley** [comment]
I'm QAing this currently.

**Comment by danielbeardsley** [comment]
QA :+1: 

Downloads now work and the files are owned by the right user (not root).

image

**Pull request merged**